akka-http-core/src/main/mima-filters/10.1.x.backwards.excludes/3187-remove-legacy-pool.excludes

```diff
@@ -0,0 +1,16 @@
+# Removal of @ApiMayChange methods
+ProblemFilters.exclude[MissingClassProblem]("akka.http.javadsl.settings.PoolImplementation$")
```

Comment:
We could have kept (part of) those methods around for a while for better binary compatibility but that might not be necessary (because people would have used config settings in most cases anyway).